client/src/test/java/io/pravega/client/segment/impl/SegmentOutputStreamTest.java

```diff
@@ -173,16 +173,15 @@ public void testConnectAndFailedSetupAppendDueToTruncation() throws Exception {
         MockController controller = new MockController(uri.getEndpoint(), uri.getPort(), cf, true);
         ClientConnection connection = mock(ClientConnection.class);
         cf.provideConnection(uri, connection);
-        @Cleanup
```

Comment:
Why this?

This object will never be cleaned up after the test.

Reply:
removed this change.

Reply:
removed this change.